crates/autopilot/src/domain/fee/mod.rs

```diff
@@ -132,6 +160,18 @@ impl ProtocolFees {
             .collect_vec();
         boundary::order::to_domain(order, protocol_fees)
     }
+
+    /// Internal function in `ProtocolFees` in order to determine whether the
+    /// driver has to apple the protocol fee to an order/solution
+    fn driver_must_apply_protocol_fee(&self, address: H160) -> bool {
```

Comment:
nit: The driver is an implementation detail that isn't really important here.
I think `owner_exempt_from_fees()` is more fitting. Obviously the internal logic would then have to be inverted to match the new name.